js/lib/ui.js

```diff
@@ -299,6 +299,7 @@ elgg.ui.requiresConfirmation = function(e) {
 	var confirmText = $(this).data('confirm') || elgg.echo('question:areyousure');
 	if (!confirm(confirmText)) {
 		e.preventDefault();
```

Comment:
This will be irrelevant if you return false. It will prevent the default stop propagation/bubbling.